monkey/infection_monkey/telemetry/credentials_telem.py

```diff
@@ -17,6 +17,9 @@ def __init__(self, credentials: Iterable[Credentials]):
         """
         self._credentials = credentials
 
+    def send(self, log_data=True):
```

Comment:
```suggestion
    def send(self, _=None):
```

Reply:
Complains, because the method signature contains a parameter named `log_data`, not named `_`

Reply:
Either way, we're implementing an interface method, we shouldn't change the interface even if we could